scripts/director-health-check.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+## Run Prerequisites
+echo -e "\n Running Prerequisites  *********************************************************************\n";
+
+echo -e "\n********* [ Making logs directory ] **********\n";
+mkdir -p logs/director-health-checks -v
+
+echo -e "\n**** [ Cloning OEP test-suite directory ] ****\n";
+git clone https://github.com/mayadata-io/oep.git
```

Comment:
@shiveshabhishek yes we need to clone it here again. We can reuse the previously cloned `oep-e2e` repo but that will need major refactoring in the scripts, so I preferred to keep it here too.

Reply:
@harshshekhar15 since we are using prerequisites now. So, IMO just updating that file will suffice. I don't think cloning twice the same thing twice is a good idea.

Reply:
Can you check once. Because in the case of konvoy, I don't see any major change.

Reply:
@shiveshabhishek no just renaming the repo will not work, we will have to change the path of the `litmus-jobs` being called under each `director-health-check` script.

Reply:
And some other minor refactors will also be needed. So to sum it up we will have to refactor each script present under `director-health-checks` directory which IMO is not ideal right now.

Reply:
Ok, I guess I wasn't able to make you understand. What I am trying to say here is: the very first time when we are cloning the `oep-e2e` repo at this [line](https://github.com/mayadata-io/oep-e2e-rancher/blob/268a80427649d98c4834ac29cd40ca194a71d529/stages/infra-setup/pre-requisites#L14) in the prerequisite, can't we clone there as `git clone https://github.com/mayadata-io/oep.git` and update the next few lines with `oep` instead of `oep-e2e` in the prerequisite file? In that case, the scripts under `director-health-check` need not be changed. And you can remove the repo cloning command from there.

Reply:
@shiveshabhishek I got you the very first time. Ok let me keep it this way -- every script under `director-health-checks` directory runs some litmus jobs which is being applied from the `oep` repo that is being cloned here. If we want to use the initially cloned repo we will have to modify the path of the each litmus job being applied in each of the scripts.

Reply:
Inshort there is some path dependencies that needs to be resolved in each of the script in order to reuse the same repo.

Reply:
@shiveshabhishek if we will reuse the previous repo we will have to modify all these paths in each of the scripts --
- https://github.com/mayadata-io/oep-e2e/blob/add-scripts/scripts/director-health-checks/alertmanager-check.sh#L11